deploy/clowdapp.yaml

```diff
@@ -211,6 +211,8 @@ objects:
               name: vulnerability-slack
         - name: VMAAS_ENV
           value: ${VMAAS_ENV}
+        - name: PROMETHEUS_MULTIPROC_DIR
```

Comment:
@jdobes @yungbender Shouldn't this env be also set on reposcan?

I'm getting traceback in ephemeral:
```
Traceback (most recent call last):
  File "/usr/lib64/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib64/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/vmaas/vmaas/reposcan/main.py", line 8, in <module>
    from vmaas.reposcan.reposcan import create_app, DEFAULT_PATH, DEFAULT_PATH_API
  File "/vmaas/vmaas/reposcan/reposcan.py", line 33, in <module>
    from vmaas.reposcan.mnm import ADMIN_REQUESTS, FAILED_AUTH, FAILED_IMPORT_CVE, FAILED_IMPORT_CPE, OVAL_FAILED_IMPORT,\
  File "/vmaas/vmaas/reposcan/mnm.py", line 15, in <module>
    multiprocess.MultiProcessCollector(REGISTRY)
  File "/usr/local/lib/python3.6/site-packages/prometheus_client/multiprocess.py", line 33, in __init__
    raise ValueError('env PROMETHEUS_MULTIPROC_DIR is not set or not a directory')
ValueError: env PROMETHEUS_MULTIPROC_DIR is not set or not a directory
```

Reply:
Never mind. It is exactly where this adds it. There must have been some OCP & bonfire mess up.